core/src/main/scala/com/raphtory/api/analysis/graphstate/Accumulator.scala

```diff
@@ -8,7 +8,7 @@ package com.raphtory.api.analysis.graphstate
   *
   * @see [[GraphState]]
   */
-trait Accumulator[-S, T] {
+trait Accumulator[@specialized(Int) -S, T] {
```

Comment:
Do we want this to extend beyond Int?

So @specialized(Int, Long, Boolean, ...)

Reply:
Yes, it's WIP still

Reply:
It's specialised now